testing/events.go

```diff
@@ -120,6 +120,22 @@ func ParsePacketFromEvents(events []abci.Event) (channeltypes.Packet, error) {
 	return channeltypes.Packet{}, fmt.Errorf("acknowledgement event attribute not found")
 }
 
+// ParsePacketsFromEvents parses events emitted from a MsgRecvPacket and returns
+// all the packets found.
+func ParsePacketsFromEvents(events []abci.Event) ([]channeltypes.Packet, error) {
+	var packets []channeltypes.Packet
+	for i, ev := range events {
+		if ev.Type == channeltypes.EventTypeSendPacket {
```

Comment:
I think we could also remove this `if` statement, because the same one is in `ParsePacketFromEvents`.

Reply:
done with the logic moved from one function to the other.